application/frontend/controllers/MfaController.php

```diff
@@ -115,6 +115,14 @@ public function actionVerify(int $id, string $type = "")
             throw new BadRequestHttpException();
         }
         $mfa->loadData($rpOrigin);
+        $user->last_login_utc = MySqlDateTime::now();
```

Comment:
I think this controller action is also used to verify a new webauthn, but I could be wrong.

Reply:
It looks that way from the tests. Any ideas on how to only update it when logging in? I was wondering about only updating if the previous one is older than the token lifespan.